DEVELOPING_TUTORIALS.md

```diff
@@ -639,48 +562,40 @@ Note that your user history on the live website (https://proto.school) is differ
 
 ### Renaming a tutorial after it has been published
 
-On rare occassion, we may need to change the name of a tutorial, and its related URL, after it has been published. Because we use IDs as our primary point of reference and have built a migration tool for the cache, it's possible to do this without affecting the user's progress indicators.
+1. In the `src/static/tutorials.json` file, rename `title` to the new desired title.
+1. Add old tutorial `url` value to the `redirects` attribute. (e.g. `"redirects": ["old-tutorial-short-title"]`)
+1. Update `url` with the new name.
+1. Rename folder `src/tutorials/xxxx-old-tutorial-short-title` to the new url. (e.g. `src/tutorials/xxxx-new-tutorial-short-title`)
 
-To change a tutorial's name, go to its entry in `static/tutorials.json` and update its `title` and `url`. Remember that the URL you use will be converted into a shortened title displayed at the top of each lesson. (For example, the URL `short-tutorial-title` would create the header  `Short Tutorial Title | Lesson 2 of 3` above the title for its second lesson.)
+That's it! Next time you run ProtoSchool the tutorial should be renamed and users will still have access to the status of their lesson progress. If a user tries to access the old URL for your tutorial or one of its lessons, they'll be redirected to the tutorial landing page at its new URL.
 
-```js
-"0005": {
-  "url": "new-short-tutorial-title",
-  "project": "IPFS",
-  "title": "New name of this tutorial",
-  "description": "Description of tutorial",
-  // etc.
-```
+### My files (.md, -exercise.md) content are not showing up in the lesson page
 
-Then in the `src/main.js` file, find the routes for your tutorial by its ID (which will remain unchanged) and update the paths to reflect the new `url` value in `static/tutorials.json`:
+The file might not be loading correctly. You can use the `DEBUG=true` option to get warnings in the console. To enable it run:
 
-```js
-// Tutorial 0005
-{ path: '/new-short-tutorial-title', component: Landing, props: { tutorialId: '0005' } },
-{ path: '/new-short-tutorial-title/resources', component: ResourcesLesson, props: { tutorialId: '0005' } },
-{ path: '/new-short-tutorial-title/01', component: T0002L01, props: { tutorialId: '0005', lessonId: '01' } },
-{ path: '/new-short-tutorial-title/02', component: T0002L02, props: { tutorialId: '0005', lessonId: '02' } },
-{ path: '/new-short-tutorial-title/03', component: T0002L03, props: { tutorialId: '0005', lessonId: '03' } },
+```sh
+DEBUG=true npm start
 ```
 
-In case someone uses an old link shared with them before the tutorial name and URL were changed, you'll also need to add two redirect objects as follows.
+Check the browser dev tools and check for warning logs.
```

Comment:
```suggestion
Check your browser dev tools for warning logs.
```

Reply:
What's the most likely culprit? A misnamed file?

Reply:
Yeah, misnamed file or located in the wrong directory.